obs/project.go

```diff
@@ -40,13 +40,98 @@ type Project struct {
 	UseForBuild  *UseForBuild `json:"useForBuild,omitempty" xml:"useforbuild,omitempty"`
 }
 
-type Client struct {
-	Client   http.Client
+// OBS is the wrapper struct
```

Comment:
Please add proper comments, see https://github.com/kubernetes-sigs/release-sdk/blob/main/github/github.go for inspiration

```suggestion
// OBS is a wrapper around OBS related functionality
```